resources/aux/keycloak-yml-template

```diff
@@ -0,0 +1,9 @@
+version: '3.1'
```

Comment:
I believe it should be `keycloak-template.yml`, shouldn't it @cecchisandrone?

Reply:
see file changes @raxkaynan

Reply:
I can see that there's also `Dockerfile-template` and `gitignore-template`. Quite inconsistent if some are named `default` and others `template`. What do you think @cecchisandrone?

Reply:
renamed every template prefix to `default`

Reply:
ok